sky/data/storage.py

```diff
@@ -1001,7 +1001,25 @@ def warn_for_git_dir(source: str):
                 else:
                     for source in self.source:
                         warn_for_git_dir(source)
+            # init logger on demand
+            storage_logger = sky_logging.init_logger(
+                sky_logging.STORAGE_LOGGER_NAME)
+            log_path = sky_logging.get_logger_log_file_path(
```

Comment:
Can we show the path next to the spinner when it's running, like we have for file_mounts sync?
```
⠇ Syncing ~/tmp-workdir -> s3://romilbuckettest5/  [Dim] View logs at ....
```